src/js/visual/configurator-csr.js

```diff
@@ -1783,22 +1793,24 @@ ripe.ConfiguratorCsr.prototype._registerInitialsHandlers = function() {
  * @ignore
  */
 ripe.ConfiguratorCsr.prototype._unregisterInitialsHandlers = function() {
-    this.owner && this.owner.unbind("initials_extra", this._onInitialsExtraEvent);
-    this.owner && this.owner.unbind("initials", this._onInitialsEvent);
+    this.owner && this.owner.unbind("initials_extra", this._onInitialsExtraEventCallback);
+    this.owner && this.owner.unbind("initials", this._onInitialsEventCallback);
 };
 
 /**
  * @ignore
  */
 ripe.ConfiguratorCsr.prototype._registerConfigHandlers = function() {
-    this.owner.bind("pre_config", (brand, model, options) => this._onPreConfig(this));
-    this.owner.bind("post_config", config => this._onPostConfig(this, config));
+    // unregister previous config handlers to avoid duplicate handlers
+    this._unregisterConfigHandlers();
+    this._onPreConfigEventCallback = this.owner.bind("pre_config", (brand, model, options) => this._onPreConfig(this));
+    this._onPostConfigEventCallback = this.owner.bind("post_config", config => this._onPostConfig(this, config));

```

Comment:
Make the name shorter, like `_onPreConfigBind`, and add the variable declaration to here: https://github.com/ripe-tech/ripe-sdk/blob/b327d752b9578f542abace9612f4276d7dc6fc27/src/js/visual/configurator-csr.js#L95